AnkiDroid/src/main/java/com/ichi2/anki/preferences/DevOptionsFragment.kt

```diff
@@ -86,6 +89,25 @@ class DevOptionsFragment : SettingsFragment() {
             launchCatchingTask { CollectionManager.withCol { Thread.sleep(1000 * 86400) } }
             false
         }
+        // Make it possible to test crash reporting
+        requirePreference<Preference>(R.string.pref_set_database_path_key).setOnPreferenceClickListener {
+            AlertDialog.Builder(requireContext()).show {
+                setTitle("Warning!")
+                setMessage("This will most likely make it so that you cannot access your collection. It will be very difficult to recover your data.")
+                setPositiveButton(R.string.dialog_ok) { _, _ ->
+                    Timber.w("Setting collection path to /storage/emulated/0/AnkiDroid")
+                    val prefs = AnkiDroidApp.sharedPrefs()
+                    prefs.edit {
+                        putString(
+                            CollectionHelper.PREF_COLLECTION_PATH,
+                            "/storage/emulated/0/AnkiDroid"
+                        )
+                    }
+                }
+                setNegativeButton(R.string.dialog_cancel) { _, _ -> }
```

Comment:
I **think** you can omit the lambda

```suggestion
                setNegativeButton(R.string.dialog_cancel)
```

Reply:
No can do